bin/mmdb_utils.py

```diff
@@ -47,6 +60,15 @@
 
 
 
+def convert_to_bool_default_true(val):
+    if not val:
```

Comment:
do we need if? it will be true by default

Reply:
Done